modules/e2ee/E2EEncryption.js

```diff
@@ -130,6 +130,12 @@ export class E2EEncryption {
 
         this._enabled = enabled;
 
+        if (enabled) {
```

Comment:
Let's protect this function from being called multiple times with different values by using some `enabling` Deferred and throwing an "operation in progress" exception. In the UI we will want to disable the control while this function resolves.